src/lib/components/forms/picture-input.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+  import Icon from '$lib/components/icons/icon.svelte';
+  import Icons from '$lib/components/icons/icons';
+
+  export let text: string;
+  let avatar, fileinput;
```

Comment:
Don't forget the typescript types! :)
```suggestion
  let avatar: string;
  let fileinput: HTMLInputElement;
```